IntegrationTests/tests_04_performance/test_01_resources/test_jsonvalue_to_bytes.swift

```diff
@@ -1,17 +1,19 @@
 import PureSwiftJSONParsing
 
 func run(identifier: String) {
-    let sampleString = SampleStructure.sampleJSON
-    let sampleBytes  = [UInt8](sampleString.utf8)
-    let sampleJSON   = try! JSONParser().parse(bytes: sampleBytes)
-    
-    measure(identifier: identifier) {
-        for _ in 0..<1_000 {
-          var bytes = [UInt8]()       // <-- this is the only real allocation
-          bytes.reserveCapacity(6000) // <-- reserve capactity so high we don't need reallocs
-          sampleJSON.appendBytes(to: &bytes)
-        }
-      
-        return 1_000
+  let sampleString = SampleStructure.sampleJSON
+  let sampleBytes  = [UInt8](sampleString.utf8)
+  let sampleJSON   = try! JSONParser().parse(bytes: sampleBytes)
+
+  var bytes = [UInt8]()       // <-- this is the only real allocation
+  bytes.reserveCapacity(6000) // <-- reserve capactity so high we don't need reallocs
+  
+  measure(identifier: identifier) {
+    for _ in 0..<1_000 {
+      bytes.removeAll(keepingCapacity: true)
+      sampleJSON.appendBytes(to: &bytes)
     }
+  
+    return 1_000
```

Comment:
tiny nit so the optimiser is definitely not able to optimise the whole loop away (it won't be but still)... Instead of `return 1_000` do `return bytes.count`. It's good to return some number that is "calculated" by the actual code to make sure the code actually has to run

Reply:
Fixed! Thanks a lot!